src/components/Emoticon/Emoticon.css.js

```diff
@@ -0,0 +1,81 @@
+import styled from '../styled'
+
+export const reactionEmoticonsColours = {
+  off: {
+    head: '#E5E9EC',
+    face: '#A5B2BD',
+  },
+  on: {
+    face: '#2E2D2E',
+    head: {
+      happy: '#FFDA6A',
+      okay: '#D2D2D2',
+      sad: '#BDD6FF',
+    },
+  },
+}
```

Comment:
I see you are using colours outside of our palette here, except #A5B2BD which happens to be grey.800. I do wonder, if we should leave a comment here that we are using custom colours so no one comes along like I just did to check them against our color palette thinking we should have used getColor here to select these values from our palette. Thoughts?